scripts/release/packager/download-dependency.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+set -euo pipefail
+
+function step { >&2 echo -e "\033[1m\033[36m* $@\033[0m"; }
+function finish { >&2 echo -en "\033[0m"; }
+trap finish EXIT
+
+NAME=${1}
```

Comment:
This is only used in the step description now. Shall we remove it?

Reply:
Yeah I'm fine with that.

Reply:
Addressed.